VideoWeb/VideoWeb/ClientApp/src/app/security/idp-selection/idp-selection.component.ts

```diff
@@ -4,32 +4,73 @@ import { Logger } from 'src/app/services/logging/logger-base';
 import { pageUrls } from '../../shared/page-url.constants';
 import { SecurityConfigSetupService } from '../security-config-setup.service';
 import { IdpProviders } from '../idp-providers';
+import { LaunchDarklyService, FEATURE_FLAGS } from 'src/app/services/launch-darkly.service';
 
 @Component({
     selector: 'app-idp-selection',
     templateUrl: './idp-selection.component.html'
 })
 export class IdpSelectionComponent {
     identityProviders = {};
+    identityProvidernames = [];
 
     selectedProvider: IdpProviders;
     submitted = false;
 
-    constructor(private router: Router, private logger: Logger, private securityConfigSetupService: SecurityConfigSetupService) {
-        this.identityProviders[IdpProviders.ejud] = {
-            url: '/' + pageUrls.Login
-        };
+    constructor(
+        private router: Router,
+        private logger: Logger,
+        private securityConfigSetupService: SecurityConfigSetupService,
+        private ldService: LaunchDarklyService
+    ) {
+        this.ldService.getFlag<boolean>(FEATURE_FLAGS.ejudiciarySignIn).subscribe(value => {
+            if (value) {
+                this.identityProviders[IdpProviders.ejud] = {
+                    url: '/' + pageUrls.Login
+                };
+                this.updateProviderNames();
+            }
+        });
+
+        this.ldService.getFlag(FEATURE_FLAGS.dom1SignIn).subscribe(featureEnabled => {
+            if (featureEnabled) {
+                this.identityProviders[IdpProviders.dom1] = {
+                    url: '/' + pageUrls.Login
+                };
+                this.updateProviderNames();
+            }
+        });
+        // this.ldService.flagChange.subscribe(value => {
```

Comment:
some commented out code here - can we remove it if it's not needed?

Reply:
done, lost a few commits along the way. the component is correct now